canonical-web-and-design/ubuntu.com
# Build da-vmware-a-charmed-openstack engage page and takeover

## Done

- Build engage page
- Build takeover
- Add to 3 pages
- Create social banner

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/engage/it/da-vmware-a-charmed-openstack and check content matches [copy doc](https://docs.google.com/document/d/1rFEOD2_VVsszbQ_hq7PRmhIwdTlhQpRHmrk9ISYhS3s/edit) and [brief](https://docs.google.com/spreadsheets/d/1Vi9svRMMN7Rh_34ezUMIdxVXld5t1ftOOFwV2E0ofno/edit#gid=1271849439)
- Check [social banner](https://tools.demos.haus/?title=Da+VMware+a+Charmed+Openstack&subtitle=Come+la+migrazione+pu%C3%B2+tagliare+i+costi&illustration_url=https%3A%2F%2Fassets.ubuntu.com%2Fv1%2F2d935f28-openstack-cloud.svg&background=dark#banner-form-modal)


## Issue / Card

Fixes #3174
